modules/parachains/src/lib.rs

```diff
@@ -91,6 +91,8 @@ pub mod pallet {
 		BoundedStorageValue<<T as Config<I>>::MaxParaHeadDataSize, ParaStoredHeaderData>;
 	/// Weight info of the given parachains pallet.
 	pub type WeightInfoOf<T, I> = <T as Config<I>>::WeightInfo;
+	type GrandpaPallet<T, I> =
```

Comment:
hmm, this is little bit tricky, it assumes that the same Instance1...N is used for pallet_bridge_parachains and its relevant pallet_bridge_grandpa,

this could be problem for example:
`pallet_bridge_grandpa::<Instance1>`
`pallet_bridge_parachains::<Instance1>` -> needs `pallet_bridge_grandpa::<Instance1>`
`pallet_bridge_parachains::<Instance2>` -> needs also `pallet_bridge_grandpa::<Instance1>`

so mayvbe we should better introduce explicit `GrandpaPalletInstance` -> `type GrandpaPallet<T, GI> =`